src/app/core/utils/api-token/api-token.service.ts

```diff
@@ -48,8 +55,12 @@ export class ApiTokenService {
     private cookiesService: CookiesService,
     private router: Router,
     private store: Store,
+    oAuthService: OAuthService,
+    oAuthConfigurationService: OAuthConfigurationService,
     appRef: ApplicationRef
   ) {
+    oAuthConfigurationService.config$.subscribe(config => oAuthService.configure(config));
```

Comment:
If I run this with a configured `type: 'auth0'` identity provider, our `OAuthService` lib cannot get the initial discovery document loaded. It seems as though the different closure contexts are mixed up. One `oAuthService` uses ICM token endpoint, another is using whatever is configured with the `type: 'auth0'` client.